### ISSUE TITLE
Is it possible to put the spinner into the second terminal column?

### ISSUE THREAD

**Issue by Tyrn** [SirWindfield/terminal-spinners-rs#8]
Hi,

Is it possible to influence the location of the spinner? I checked `lib.rs`, didn't find a clue.

UPD

I've got the desired effect editing two lines starting with line 147:
```
                    writeln!(stdout, " {} {}", emoji_to_write, self.text).unwrap();
                    should_stop_cycle_loop = true;
                } else {
                    write!(stdout, " {}{}", frame, self.text).unwrap();
```
adding a space at the beginning of format strings. This is not exactly what I hoped for.

**Comment by mainrs** [comment]
Would some kind of user-defined prefix be enough for your use-case?

**Comment by Tyrn** [comment]
Most certainly!

**Comment by mainrs** [comment]
Released v0.3.2
